include/distconfig.php

```diff
@@ -166,6 +166,14 @@
 
 // $config->setAlphabeticOrder(true);
 
+// By default, WebSVN loads parent path directories and then on user click other,
+// This options loads the entire directory in one go and allows to browse without delay.
+// By default all will be collapsed to root directory and can be expanded.
+// This superseeds the setAlphabeticOrder and everything will be shown in alphabetic 
+// order by default.
```

Comment:
This is an example how it looks. It is alphabetically sorted by default as shown.
![image](https://user-images.githubusercontent.com/4973182/92430756-507c8800-f1b3-11ea-8b75-4d0fce123850.png)

When directory is expanded then the files and folders under it are also alphabetically arranged by default.
![image](https://user-images.githubusercontent.com/4973182/92430804-7a35af00-f1b3-11ea-99e6-be79b3b60262.png)

Yes removed the sorting call from when loadAllRepos is shown. But it retains the behavior when it is false.

Reply:
OK, so when you talk about alphabetic sort order, you always mean that grouping of dirs and files is NOT supported anymore. In that case please add an additional sentence to tell so in the docs of your config. This can be improved later in the worst case and is known until then.

> // The alphabatecal order is applied to all directory and files. 

vs.

> // The alphabetical order is applied to all directory and files. This means that grouping of all dirs together and all files together is NOT supported currently!

Reply:
Wrote something like the following. 
```
// By default, WebSVN loads parent path directories and then on user click other,
// This options loads the entire directory in one go and allows to browse without delay.
// By default all will be collapsed to root directory and can be expanded.
// The performance will be impacted as it takes time to load up all the things in the 
// repository. Once loaded directory exapansion is instantaneous.
// The alphabetical order is applied to all directory and files. 
// This means that grouping of all dirs together and all files together is NOT supported currently!
// The files and directories are shown as is with a mixture of files and folders.
```